### PR TITLE
Refs [TASK] [34640] [Create UI Air Polution Screen]

### PR DESCRIPTION
**Pull request by I-am-Lien**
Refs [TASK] [34640] [Create UI Air Polution Screen]

## Related Tickets
- [#Task 34640: c](https://edu-redmine.sun-asterisk.vn/issues/34640)
## WHAT
- https://edu-redmine.sun-asterisk.vn/issues/34640
## Evidence (Screenshot or Video)
Screenshot_1
Screenshot_2

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

**Review by tuannm-1276 — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -41,11 +41,13 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    implementation 'androidx.wear:wear:1.0.0'
```
Comment (huytq-2422) [inline app/build.gradle]: ơ cái gì đấy :v

File: app/src/main/res/layout/layout_air_quality_description.xml [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+        app:layout_constraintGuide_percent="0.3" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guidelineVertical70"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.7" />
+
+
```
Comment (huytq-2422) [inline app/src/main/res/layout/layout_air_quality_description.xml]: remove 2 empty lines

File: app/src/main/res/values-round/strings.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="hello_world">Hello Round World!</string>
+</resources>
```
Comment (huytq-2422) [inline app/src/main/res/values-round/strings.xml]: sao lại khai báo trong này thế

File: app/build.gradle (hunk 1 above)
Comment (thonc-2135) [reply app/build.gradle]: chắc là dùng cho đồng hồ thông minh. =))

**Review by datdt-1935 — APPROVED** [review]

**Pull request merged**